src/modules/LanItems.rb

```diff
@@ -1148,19 +1152,19 @@ def Read
       NetworkInterfaces.CleanHotplugSymlink
 
       interfaces = getNetworkInterfaces
+      items = LanItems.Items
       # match configurations to Items list with hwinfo
       interfaces.each do |confname|
-        @Items.each do |key, value|
+        items.each do |key, value|
           match = value.fetch("hwinfo", {}).fetch("dev_name", "") == confname
-          @Items[key]["ifcfg"] = confname if match
+          items[key]["ifcfg"] = confname if match
         end
       end
 
       interfaces.each do |confname|
-        next if @Items.keys.any? { |key| @Items.fetch(key, {}).fetch("ifcfg", "") == confname }
+        next if items.keys.any? { |key| items.fetch(key, {}).fetch("ifcfg", "") == confname }
```

Comment:
this looks strange to use keys, when in fact you use key only to accept value. What about
```
next if items.values.any? { |item| item && item["ifcfg"] == confname }
```

Reply:
but just NP

Reply:
a consequence of an earlier more less direct rewrite from ycp -> ruby ... going to update ;-)